### ISSUE TITLE
Diagram on HTMLFormElement page

### ISSUE THREAD

**Issue by twrkco** [mdn/content#20589]
### MDN URL

https://developer.mozilla.org/en-US/docs/Web/API/HTMLFormElement

### What specific section or headline is this issue about?

Diagram under main section

### What information was incorrect, unhelpful, or incomplete?

It's not clear what information or relationships this diagram is attempting to convey. If, as the properties section denotes, the HTMLFormElement interface also inherits properties from its parent, HTMLElement, that's not clear at all from the diagram. What do the arrows mean, and why are the going from right to left?

<img width="516" alt="diagram" src="https://user-images.githubusercontent.com/2061889/189539978-4db4be7a-f30b-4014-96b6-fd4f54678ec5.png">

### What did you expect to see?

A blurb, citation, or aside about what the diagram means, what information it's trying to convey, or some type of explanation of the relationships between the interfaces. If it's a parent/child relationship then the diagram makes no sense, as the arrows are pointing the wrong direction.

### MDN metadata

<details>
<summary>Page report details</summary>

* Folder: `en-us/web/api/htmlformelement`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/API/HTMLFormElement
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/api/htmlformelement/index.md
* Last commit: https://github.com/mdn/content/commit/277e5969c63b97cfb55ab4a0e612e8040810f49b
* Document last modified: 2022-09-09T05:14:42.000Z

</details>

**Comment by Andrew-Cottrell** [comment]
> What do the arrows mean, and why are they going from right to left?

If the diagram was flipped horizontally, it would convey the exact same information and the arrows would be pointing left-to-right. The significant aspect is each arrow's orientation between pairs of interfaces: why do they point from child interface to parent interface?

The diagram denotes inheritance via the prototype chain. The arrows can be understood to represent the pointers (or references) from each interface's prototype to its parent interface's prototype stored as the value of the internal `[[Prototype]]` slot, which is returned by the [`Object.getPrototypeOf`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/getPrototypeOf) method.

```javascript
console.log(Object.getPrototypeOf(HTMLFormElement.prototype).constructor.name); // "HTMLElement"
console.log(Object.getPrototypeOf(    HTMLElement.prototype).constructor.name); // "Element"
console.log(Object.getPrototypeOf(        Element.prototype).constructor.name); // "Node"
console.log(Object.getPrototypeOf(           Node.prototype).constructor.name); // "EventTarget"
```

> If it's a parent/child relationship then the diagram makes no sense, as the arrows are pointing the wrong direction.

In simple terms, the arrows mean "_inherits from_". The direction of each arrow indicates that a child interface _inherits from_ a parent interface. If the arrows instead meant "_inherited by_", then they would point the other way.
